### ISSUE TITLE
[P2][Sourcing][Accessibility] Not able to reach freeze/unfreeze buttons. On clicking freezable columns using keyboard, it is automatically freezing/unfreezing.

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#11612]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Continuation of https://github.com/SAP/fundamental-ngx/issues/11578
Expectation: the column should not freeze , until and unless user opens menu option and then presses ENTER key

#### Can you handle that on the application side
No 

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
Angular 15
core / platform: 0.43.29

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Press enter on Column header and see that its freezes the column
The user is not even moving to menu option and beforehand itself the column is frozen
Expectation: the column should not freeze , until and unless user opens menu option and then presses ENTER key

#### Please provide stackblitz example(s).
Pls refer https://github.com/SAP/fundamental-ngx/issues/11578

**Comment by sagar7259** [comment]
Hello,

the column headers should be activated by both spacebar and enter key to get the menu item 'Freeze/unfreeze column.
There are two different behaviours here

When enter key is used for this a menu will popup which has an item 'Freeze column', then user can select this to freeze a column.
But for the same when spacebar is used, the column is getting frozen directly without showing any popup menu here.
The expected behaviour is that both enter and spacebar key should behave in the same manner.

Can you confirm the behaviour which is right..?

Regards,
Sagar

Regards,
Sagar

**Comment by droshev** [comment]
@nikolay-kolarov Can you advise us here?

**Comment by droshev** [comment]
@I543348 The column freezes only when you open the menu and you press ENTER, if you press escape then the menu closes and the column is not frozen

**Comment by I543348** [comment]
> @I543348 The column freezes only when you open the menu and you press ENTER, if you press escape then the menu closes and the column is not frozen

Hi @droshev , its replicable here, can you pls check ?
Looks like when column has only and only "Freeze" option in menu then the issue occurs.
Pls also see attached recording from same stackblitz link , version evaluated is 0.43.32


 https://stackblitz.com/edit/angular-9qxxca-ac9we5?file=src%2Fapp%2Fplatform-table-preserved-state-example.component.html,src%2Fapp%2Fplatform-table-preserved-state-example.component.ts,package.json 

https://github.com/SAP/fundamental-ngx/assets/88182358/cd648d6c-faaa-4830-9ea4-5b1e06c7ef51

**Comment by mikerodonnell89** [comment]
@droshev I am seeing this issue as well
